docs/resources/observability_project.md

```diff
@@ -0,0 +1,87 @@
+---
+page_title: "Elastic Cloud: ec_observability_project Resource"
+description: |-
+  
+---
+
+# Resource: ec_observability_project
+
+
+
+## Example Usage
+
+```terraform
+resource "ec_observability_project" "my_project" {
+  name      = "my_project"
+  region_id = "aws-us-east-1"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) Descriptive name for a project.
+- `region_id` (String) Unique human-readable identifier for a region in Elastic Cloud.
+
+### Optional
+
+- `alias` (String) A custom domain label compatible with RFC-1035 standards. Derived from the project name by default.
+- `optimized_for` (String) The purpose for which the hardware of this elasticsearch project is optimized for. Also known as the Elasticsearch project subtype.
+- `search_lake` (Attributes) Configuration for entire set of capabilities that make the data searchable in Elasticsearch. (see [below for nested schema](#nestedatt--search_lake))
```

Comment:
`make docs-generate` removes both `optimized_for` and `search_lake` from here as well as from `security_project.md`. It also makes some other changes.